### PR TITLE
PaymentRequest.p.paymentAddress → shippingAddress

### PR DESCRIPTION
**Pull request by sideshowbarker**
This change renames the BCD `paymentAddress` subfeature of the `PaymentRequest` feature to `shippingAddress`.

As noted in https://github.com/mdn/browser-compat-data/issues/7668, it’s been `PaymentRequest.prototype.shippingAddress` “for years, probably from the start”.

---

cc @marcoscaceres for confirmation

### CODE REVIEW

File: api/PaymentRequest.json
Comment (saschanaz) [inline api/PaymentRequest.json:623]: This note is now invalid:

```suggestion
              "flags": [
                {
                  "type": "preference",
                  "name": "dom.payments.request.enabled"
                }
              ]
```

File: api/PaymentRequest.json
Comment (saschanaz) [inline api/PaymentRequest.json:639]: Safari support should probably be 11.1 (the version that first enabled Payment Request), as it seems it has have the attribute since the beginning.

https://webkit-search.igalia.com/webkit/source/Source/WebCore/Modules/paymentrequest/PaymentRequest.idl

File: api/PaymentRequest.json
Comment (sideshowbarker) [inline api/PaymentRequest.json:627]:
```suggestion
              "notes": "Available only in nightly builds as <code>shippingAddress</code>."
```
```suggestion
```

File: api/PaymentRequest.json
Comment (sideshowbarker) [reply api/PaymentRequest.json:643]: > Safari support should probably be 11.1 (the version that first enabled Payment Request), as it seems it has have the attribute since the beginning.
> 
> https://webkit-search.igalia.com/webkit/source/Source/WebCore/Modules/paymentrequest/PaymentRequest.idl

Thanks — will raise a separate PR for that

**Review by ddbeck — APPROVED** [review]
This looks good to me.

Do you want to wait for Marcos's confirmation before merging?

**Comment by foolip** [comment]
Marcos has an away status, so I would suggest not waiting.

**Pull request merged**
